src/common/TALENTS/deathknight.ts

```diff
@@ -245,7 +245,7 @@ const talents = {
   CHILL_STREAK_TALENT: {
     id: 305392,
     name: 'Chill Streak',
-    icon: 'spell_frost_piercing_chill',
+    icon: 'spell_frost_piercing-chill',
```

Comment:
can add this to the `BAD_ICONS` object to make it persist across regens: https://github.com/WoWAnalyzer/WoWAnalyzer/blob/b67a4cbb5d48d98450accd2dbf6e9e34a3c214c4/src/interface/BAD_ICONS.ts#L11-L16